CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ without shipping useless bits. TBD!
 - Support for `react@0.14.x` has been dropped, although _our_ tests pass
 this is likely to break in other apps. We now require `react@^0.14.9`, or
 anything on the `15.x` branch
+- `<RequireForAccess/>` now only accepts a *single* child node
+
+### Removed
```

Comment:
Although there's fewer divs in the output, I wouldn't think of this as a feature removal per se.